module-vfs/logdump/include/logdump/logdump.h

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2017-2021, Mudita Sp. z.o.o. All rights reserved.
+// For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+#pragma once
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+    /// @return: < 0 - error occurred during log flush
+    /// @return:   0 - log flush did not happen
+    /// @return:   1 - log flush successful
+    int dumpLogs();
```

Comment:
Is it ok that log dump functionality is a part of module-vfs and not a logger itself (with needed dependencies injected)?
Actually, the only dependency needed here is a dump file path.

Reply:
moved to logger

Reply:
still believe that neither logger nor vfs are proper places to put this. IMHO, it should be moved to some product/family of products common place. However, lets not do it now - there is more things to clean up